### ISSUE TITLE
[screenreader] Pressing continue doesn't read new page or set focus to the top of the page

### ISSUE THREAD

**Issue by nkkl** [18F/cms-hitech-apd#205]
Describe the task here, including any supplemental information.

### Repro steps
1. Turn on VoiceOver (or other screenreader)
2. Open [the app](https://hitech-apd.app.cloud.gov/state-start)
3. Use the keyboard to navigate to the `continue` button
4. Use `command` + `option` + `➡` to activate the `continue` button to move to the next section

Expected:
New section loads and the section name is announced (in this case, "Review your state contact information"). Focus is set to the top of the page so that the description ("We already have some information about Vermont from our records.") is read when `command` + `option` + `➡` is triggered.

Actual:
New section loads silently, giving no indication that the content has changed. Focus remains on the `continue` button.

### This task is done when...
 - [ ] Changes to the content on the page are announced by the screenreader
 - [ ] Focus is moved to the top of the form after activating `continue`

**Comment by brendansudol** [comment]
we've moved away from this UI
